mmv1/third_party/terraform/fwprovider/framework_provider.go.tmpl

```diff
@@ -295,6 +295,7 @@ func (p *FrameworkProvider) DataSources(_ context.Context) []func() datasource.D
         resourcemanager.NewGoogleClientConfigDataSource,
         resourcemanager.NewGoogleClientOpenIDUserinfoDataSource,
         {{- if ne $.TargetVersionName "ga" }}
+        resourcemanager.NewGoogleProjectFWDataSource,
```

Comment:
do we want to make this datasource test-only for now? like in [`a8c3757` (#11599)](https://github.com/GoogleCloudPlatform/magic-modules/pull/11599/commits/a8c375712b0404a2fd409943d82c92cd0aabbe14#diff-7c30a2afcc18eb3bf49388e586c1bcb0f84fcb5768c5a9366d4d34f0dd9dd548R443)